### PR TITLE
Feature/rs 88

### PR DESCRIPTION
**Pull request by jj-perry**

### CODE REVIEW

File: src/features/establishmentContactDetails.feature [hunk 1]
```diff
@@ -105,12 +105,12 @@ Feature: As Samantha I need to be able to fill in the establishment contact deta
...
         And I expect that element "estabContactDetails.primaryPhoneNumber" contains the text "01234567890"
-        When I add "addition" to the inputfield "estabContactDetails.emailAddress"
-        And I add "addition" to the inputfield "estabContactDetails.primaryPhoneNumber"
+        When I set "different@email.com" to the inputfield "estabContactDetails.emailAddress"
+        And I set "09876543210" to the inputfield "estabContactDetails.primaryPhoneNumber"
         And I click on the element "estabContactDetails.reuseButton"
         And I click on the element "estabContactDetails.reuseButton"
-        Then I expect that element "estabContactDetails.emailAddress" not contains the text "addition"
-        And I expect that element "estabContactDetails.primaryPhoneNumber" not contains the text "addition"
+        Then I expect that element "estabContactDetails.emailAddress" not contains the text "different"
```
Comment (paul-burnett) [inline src/features/establishmentContactDetails.feature:112]: This should be different@email.com

File: wdio.conf.js [hunk 2]
```diff
@@ -254,7 +254,8 @@ const initSeleniumConfig = (isLocal, config = {}) => {
 
 const initBrowserStackConfig = (isLocal, config = {}) => {
   let mode = MODE_BROWSERSTACK;
-  let isCi = process.env.IS_CI !== "";
+  //let isCi = process.env.IS_CI !== "";
```
Comment (paul-burnett) [inline wdio.conf.js:257]: Personally I would remove the old commented code in this file (line 257 and 296-329, leave the bits in the middle), I don't think it adds anything to leave them here, and it wouldn't be hard to reinstate the IS_CI check if it turns out it is required for something.

File: wdio.conf.js [hunk 3]
```diff
@@ -269,40 +270,63 @@ const initBrowserStackConfig = (isLocal, config = {}) => {
...
     config.capabilities = [
       capabilityFirefox(mode),
       capabilityChrome(mode),
       capabilitySafari(mode),
       capabilityEdge(mode),
-      capabilityIE(mode),
-      capabilityiOS(mode),
-      capabilityAndroid(mode),
+      //capabilityIE(mode),
+      //capabilityiOS(mode),
```
Comment (paul-burnett) [inline wdio.conf.js:280]: Presumably these two are intentionally commented out at the moment?

Comment (jj-perry) [reply wdio.conf.js:280]: Yep, not quite managed to get these ones working in time for this release

File: wdio.conf.js (hunk 2 above)
Comment (jj-perry) [reply wdio.conf.js:257]: Done

File: src/features/establishmentContactDetails.feature (hunk 1 above)
Comment (jj-perry) [reply src/features/establishmentContactDetails.feature:112]: Thanks, done.

**Pull request merged**